scylladb/scylla
# docs: add organization

@lauranovich @tzach This PR adds moves divides the dev docs into separate folders according to the organization plan.

## Before merging ...
The repo maintainer should activate GitHub pages:

1. Open the repository Settings, and scroll down to the "GitHub Pages" section.
2. Select `gh-pages` branch.
3. Wait a couple of minutes, it might take a while until GitHub applies the changes. If everything goes well, you will see the docs published under https://scylladb.github.io/scylla

## How to test changes

1. Run the command ``make preview`` from the ``docs`` directory.
3. Check the terminal where you have executed the previous command. It should not raise warnings.
3. Open in a new browser tab http://127.0.0.1:5500/ to see the new proposed organization.


## Next Steps

 In a subsequent iteration, @lauranovich and I will submit another PR moving all the [Wiki contents](https://github.com/scylladb/scylla/wiki) to the docs folder.
